jedi/_compatibility.py

```diff
@@ -161,6 +161,58 @@ def find_module_pre_py33(string, path=None, fullname=None):
 """
 
 
+def _iter_modules(paths, prefix=''):
+    # Copy of pkgutil.iter_modules adapted to work with namespaces
+
+    for path in paths:
+        importer = pkgutil.get_importer(path)
+
+        if not isinstance(importer, importlib.machinery.FileFinder):
+            for mod_info in pkgutil.iter_modules([path], prefix):
+                yield mod_info
+            continue
+
```

Comment:
Please comment that every thing from here on can be found in the function `pkgutil._iter_file_finder_modules`.